docs/pages/getting-started.mdx

```diff
@@ -184,25 +183,25 @@ Here's a selection of compatible two-factor authentication apps:
     tar -xzf teleport-v(=teleport.version=)-linux-amd64-bin.tar.gz
     cd teleport
     sudo ./install
-    Teleport binaries have been copied to /usr/local/bin
-    To configure the systemd service for Teleport take a look at examples/systemd/README.mdx
+    # Teleport binaries have been copied to /usr/local/bin
+    # To configure the systemd service for Teleport take a look at examples/systemd/README.mdx
     ```
   </TabItem>
 </Tabs>
 
-## Step 3: Log in using tsh
+## Step 3/4. Log in using tsh
```

Comment:
Visually, I find these a little strange. I wonder if we can apply a better visual style for them.